### ISSUE TITLE
Execute task with configurable role

### ISSUE THREAD

**Issue by tstroband** [OlafConijn/AwsOrganizationFormation#53]
All tasks that org-formation executes are deployed using the "OrganizationAccountAccessRole" in the target account. This "full access" role ("*:*" on "*") is created when the account is created and allows the master account to assume it. 

Customizing or restricting this role is probably not a good idea. First of all, this cannot be done through infra-as-code (CloudFormation) as the role is a pre-existing role and secondly, the chances are you're going to break the org-formation pipeline, requiring to do manual work.

However, in some cases, it is required to provision a particular stack using a different role. One such case is deploying an EKS cluster through CloudFormation. The AWS::EKS::Cluster resource has the peculiar behavior of assigning (solely) admin privileges to the role that created it. When "managing" this cluster through Lambda based custom resources the Lambda's must use the role that was used to create the cluster. As the org-formation OrganizationAccountAccessRole is not configured nor suitable to allow Lambda to use it this poses a chicken-egg problem. There are several complicated workarounds that involve creating the cluster-manage-role and then have a custom lambda use that role to create the cluster through cloud formation but this seems very cumbersome and brittle. The ability to execute a particular org-formation task with a specific role is therefore very useful.

There are other cases in which this ability is also useful. TODO: describe other cases

The most basic way of doing this through org-formation would be to allow the following syntax:

```
MyTask:
  Type: update-stacks
  Template: ./my-template.yml
  StackName: my-stack
  StackDescription: My Stack which is created with a different role
  DefaultOrganizationBinding:
    Region: eu-west-1
    Account: !Ref MyAccount
  TaskRole: my-pre-existing-role
  Parameters:
    myParam: someValue
```

The constraints of this pre-existing role are:
- The 'local' OrganizationAccountAccessRole is allowed to assume it
- The role must have enough permissions to execute cloudformation 
- ???

**Comment by OlafConijn** [comment]
hi, we added:
* `TaskRoleName`, if specified, used for cross account access to the target account when accessing the `CloudFormation` service. This taskRoleName should be usable for other tasks to.
* `CloudFormationRoleName`, if specified, passed to CloudFormation updateStack/deleteStack/etc as RoleARN attribute.

```
TestCustomRole:
  DependsOn: MyRole
  Type: update-stacks
  StackName: integration-test-custom-role
  Template: ./buckets.yml
  CloudFormationRoleName: MyCloudFormmationRole
  TaskRoleName: MyTaskRole
```

**Comment by OlafConijn** [comment]
part of ec6180817267ef65088f306e3ecc59c1da2b7e76

**Comment by OlafConijn** [comment]
implemented in version 0.9.4
